diesel/src/migration/mod.rs

```diff
@@ -0,0 +1,55 @@
+//! Representation of migrations
+
+pub mod errors;
```

Comment:
Does this need to be public or is `pub(crate)` enough?

Reply:
This needs to be public, it's re-exported from diesel_migrations